menu.html

```diff
@@ -3,6 +3,11 @@
 <head>
  <link rel="stylesheet" href="style.css">
 
+ <!--Import Google Icon Font-->
+ <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+
+ <link type="text/css" rel="stylesheet" href="css/materialize.min.css"  media="screen,projection"/>
```

Comment:
This should be: <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/css/materialize.min.css">

Reply:
Your carousel won't work until you fix this.